glide.lock

```diff
@@ -1,32 +1,119 @@
-hash: 78c31a81cf8d4aa9d05bff8041cfdded835006f3bc3cb75e398f86f015abc716
-updated: 2016-09-23T17:33:15.67392395+05:30
+hash: e4a04ec7ffb626e1ece87c0eed2c144ce3331adcbff4cc2e38984a46ada6c5e0
```

Comment:
Yeah, I hear you. etcd version changed from `v3.0.8` to `v3.1.0-rc.0`. Most of the diff you see is from new subpackages of etcd.